stdlib/xmlrpc/client.pyi

```diff
@@ -13,7 +13,7 @@ class _SupportsTimeTuple(Protocol):
     def timetuple(self) -> time.struct_time: ...
 
 _DateTimeComparable = Union[DateTime, datetime, str, _SupportsTimeTuple]
-_Marshallable = Union[None, bool, int, float, str, bytes, tuple, list, dict, datetime, DateTime, Binary]
+_Marshallable = Union[None, bool, int, float, str, bytes, Tuple[Any, ...], List[Any], Dict[Any, Any], datetime, DateTime, Binary]
```

Comment:
I wonder if this should be the same as the _Marshallable in the next file, which is missing `DateTime` and `Binary`.